tmk_core/protocol/lufa/lufa.c

```diff
@@ -1086,6 +1090,10 @@ int main(void)
         MIDI_Device_USBTask(&USB_MIDI_Interface);
 #endif
 
+#ifdef VELOCIKEY_ENABLE
+    if (velocikey_enabled()) { velocikey_decelerate();  }
```

Comment:
This should be moved to `keyboard_task`, instead of here, so that it's not hardware specific. 

https://github.com/qmk/qmk_firmware/blob/85195d7e7cbdea2c0f48fa0482dbec028c47e9ee/tmk_core/common/keyboard.c#L216